src/pages/journal/components/test-slot/test-slot.scss

```diff
@@ -1,2 +1,10 @@
 test-slot {
+  ion-card {
+    &.test-slot-portrait-mode {
```

Comment:
Is this to get 7 slots to fit without scrolling? I was under the impression we were going to allow scrolling in portrait?

Reply:
Partly, it is also to try and keep a consistent height for all portrait slots as the data varies from 1 - 3 lines depending on category. On landscape we have less room and only use 2 lines so we keep the slots smaller.